### DISCUSSION TITLE
Purchasing a Licence - 1. Credit card payment only? 2. Family licences available ?

### DISCUSSION THREAD

**Discussion by Axel-3-drei** [aclap-dev/video-downloadhelper#1107] [License & Purchase requests]
### What's your issue?

I am using vdh here in Germany since quite some time (and really like it) - finally I should like to buy me a licence.

1. Problem: Payment

This may be a surprise for people in North America: but (like so many people in Germany) I do not have a credit card. In Germany a credit card is not (as in the US) a means to demonstrate your creditability - but it only is a means of payment, just ONE means of payment, and a quite expensive one, expensive for the holder of the card as well as for the shop (normally: when a German shows up with a credit card then he/she is asked whether he/she does not have some other card or just pay cash. So I cancelled this costly (and quite useless) card after not using it for 10 years or so...)

Can you provide another means of payment ? My favourite would be Paypal.
I shall send an email to licences_at_... to give you my full email address without displaying it here.

2. Question: Family licence

I assume that your licence is for one computer (one licence to be installed on one computer only).

Can one buy something like a family licence?
(in this case: we are a couple of two retireds, no children - one desktop computer each plus our TV (consists of win11 + display), maybe plus one laptop each)

Thanks in advance for your reply and help.
Axel

### The 10 first digits (only 10) of your license number or email address

axellucas@

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
HI Axel,
Sorry there is no other way for the moment than using Stripe platform for payments. We had issues with paypal and don't use it anymore.

For your license you can use it on all your devices.

jerome

**Reply by Optyfen** [reply]
Has Stripe the ability to add payment through IBAN as a method of paying? If it has it would cure my issue, and probably also of OP.

**Comment by dasTohsy** [comment]
> ### What's your issue?
> 
> I am using vdh here in Germany since quite some time (and really like it) - finally I should like to buy me a licence.
> 
>     1. Problem: Payment
> 
> 


> 
>     * [x]  I confirm

same.

in Germany most people have no credit card. only 20% max.

i wish other payments methods .

Sorry, my english is bad.

last years i not buy an licence, because i not use credit cards. 

paypal is ok.

**Comment by Mattmjs** [comment]
@Axel-3-drei  @dasTohsy , what about using a debit card. They work in much the same way as a credit card and it uses your own
money as it connected to your bank account. Walk into your bank and see if you can get one setup.
I'd like to hear if it works and you succeed in getting a license this way.

**Reply by stereotron** [reply]
The EC card is such a debitcard but the number doesn't fit the international number system of credit cards required by the Stripe form.
Mastercard and VISA debit cards come with an annual fee and truckloads of loans, travel insurances and other stuff I don't need or want.
Virtual credit card numbers also seem not to be available here. (edit: see [Mattmjs](https://github.com/Mattmjs) reply below, Debit gift cards seem to work)

**Comment by JackJatlack** [comment]
> 1. Problem: Payment
> 
> This may be a surprise for people in North America: but (like so many people in Germany) I do not have a credit card. In Germany a credit card is not (as in the US) a means to demonstrate your creditability - but it only is a means of payment, just ONE means of payment, and a quite expensive one, expensive for the holder of the card as well as for the shop (normally: when a German shows up with a credit card then he/she is asked whether he/she does not have some other card or just pay cash. So I cancelled this costly (and quite useless) card after not using it for 10 years or so...)

I have the same problem: In Germany, I don’t need a credit card, so I don’t have one. Besides, you hardly need one here since you can pay for almost everything in stores with a barcode on your phone, fee-free. Now, the question is whether someone has already found a solution. PayPal would also be my only payment method. I would even be willing to send someone the money via PayPal if they have an alternative payment method and can purchase the license for me that way.

**Reply by Mattmjs** [reply]
I believe they don't use PayPal anymore was because years ago ( not sure when ) someone made frivolous report against
their PayPal account. PayPal locked down the account and took all of the money. I get that a lot of people want to use PayPal
but I don't think they'll ever use this payment method again. 

I don't have  & need a credit either but I had a look in ALDI where I am in Aus and there was a gift card section that had visa gift cards.

There were $50 & $100 cards, also another one that had a value from $25 to $500 which I think you choose a value in that range
at the checkout. I might even try the $50 card and see if a donation can be made.

Whether it's the same in Germany I don't know but you should look and find out.

**Reply by stereotron** [reply]
Thank you for your explanation! 
That gift card will certainly do the job as it has that 16-digit number and the expiration date.
Paying in foreign currency seems possible, although perplexity.ai warns that some online merchants prohibit intercurrency
purchase, which doesn't seem to be the case here, as you already succeeded.
It would be **quite handy** if the producers of the download tool **would have stated this in the billing section** ("_Debit gift cards may work_") for their European customers. The research took me a while.

**Reply by Mattmjs** [reply]
Well I haven't tested it out yet to see if I succeed, did you find success to purchase @stereotron .

**Reply by stereotron** [reply]
> Well I haven't tested it out yet to see if I succeed, did you find success to purchase @stereotron .

@Axel-3-drei @Mattmjs @dasTohsy @JackJatlack 
It worked for me this way: I bought a Visa debit (gift) card for $30 on vidaplayer.com with PayPal. I had to send a copy of my identity card (Personalausweis) by email. The page then forwarded me to rewarble.com where I picked the Visa anonimy option. I asked the support whether I have to use the card for the rest of the year and was told I can pause the card immediately after using it once. The card also has an option to delete. I entered the card data from rewarble.com at downloadhelper.net and received a license key. 
_Downloading without the watermark now is possible_. The 3-D secure feature was **not** necessary for the transaction as the Visa anonimy does not have this. The card now has $9,89 left so I guess the card with $25 would have been enough.

**Comment by Mattmjs** [comment]
The $50 visa gift card from ALDI worked, I gave the software a donation just in time for Christmas.

**Comment by MUPFT** [comment]
Okay, same here - I happily have survived over fifty years on this planet without any credit card and I don't intend to change that just for one meager money transfer...BUT I still would like to purchase a licence, so - any chance of me getting rid of that blasted QR code?
As PayPal isn't in it anymore, how about a good old bank transaction?
Any help really appreciated!

**Reply by jcfrog** [reply]
Hi,
Sorry, there is no other way than using Stripe payment platform for the moment.

**Reply by stereotron** [reply]
> Okay, same here - I happily have survived over fifty years on this planet without any credit card

I can relate to that very much! This instant credit availability seems unhealthy. I see only two options: purchasing a Visa gift card through vidaplayer.com (see description above) or you find someone you trust and let them make the transaction for you. Only ALDI Süd sells Visa gift cards, ALDI Nord seems not to (assuming you live in Germany).

**Reply by MUPFT** [reply]
@stereotron
...of course I do live in northern Germany - far enough from aldi Süd.
Frack - as they say in that TV Series...
Guess I have to look out for someone of those credit card maddies that I can convince to get to my aid.

**Comment by Geeblepip** [comment]
On a related note, is it possible to use iDeal to make purchases?
I checked with Stripe and they do support it, but it would have to be enabled in the merchant account. This does not come with extra fees, according to their website's customer support centre.
I have a sneaking suspicion it'd also be possible to enable direct debit payments using the same method.

I would personally really appreciate being able to use your paid-for services, and I believe the others here would, too. If you can, please let us give you our money! <3

**Comment by GrampaWildWilly** [comment]
I find this discussion fascinating.  It's so weird how the rules are so different in different countries.  I have been using credit cards for over 50 years in both Canada & the US.  I have never paid a fee to the card issuer.  I use the credit card instead of carrying cash.  I pay the bill in full on time every month.  If you see the card as a substitute for cash, then it's not dangerous.  Only irresponsible people run up huge credit card balances & end up doing something that is bad for them in the long run to get the bills paid off.  Responsibility.  That's all you need.  Be responsible about what you buy.  Having a credit card does not give you permission to live beyond your means.  It isn't just a trick to improve your credit rating.  It's a huge convenience & saves me carting cash around with me.

But this attitude is possible because pretty much every credit card has no fees.  There's competition on this.  People just don't get credit cards that have annual fees.  It's pretty slimy of the credit card companies to let us here have our cards without fees but the same companies in other countries get away with charging fees.  I don't know if this can be changed.

In any case, PayPal did VDH a dirty a number of years before I ever discovered VDH.  I never totally trusted PayPal myself.  I think I used them maybe twice & I've let it lie dormant for years.  I'm not sure I could even use it again now.

As for how VDH does business, I guess in France credit cards aren't such a rarity nor are they loaded down with fees.  I'm guessing that because VDH is based in France & the only way you can buy a VDH license is via credit card.  But I emphasize, I'm guessing.  If my guess is correct, it's just really strange that 2 countries that share a border (Germany & France) can have such different payment systems.  It's doubly strange when you realize both countries are in the euro zone & things are supposed to be like one big country over there.

I hesitate to add that if you are industrious, you can use ffmpeg to do most of your downloading instead of VDH.  But you have to be willing to learn how.  I have posted many examples of what I've called `Plan B` in this forum.  You just need to search for that, then do some studying.  I like to use VDH instead & resort to ffmpeg only when VDH really can't offer me certain features, like rewinding a livestream & putting a timer on the duration of a download & getting subtitles.  But if you can't conform to the requirements of buying a VDH license, you can always just abandon VDH & use ffmpeg.
